kolibri/plugins/device/assets/src/views/taskNotificationMixin.js

```diff
@@ -47,6 +47,8 @@ export default {
         autoDismiss: true,
         duration: 10000,
         actionText: TaskSnackbarStrings.$tr('clearTaskAction'),
+        // HACK make the snackbar appear really high to clear the bottom bar
+        bottomPosition: 80,
```

Comment:
this is an improvement.

However note that this hack doesn't account for switching between a page that has a bottom bar and one that does not